lib/unifex/code_generators/nif.ex

```diff
@@ -139,7 +139,17 @@ defmodule Unifex.CodeGenerators.NIF do
   defp generate_result_function_declaration({name, result}, ctx) do
     {_result, meta} = generate_serialization(result, ctx)
     args = meta |> Keyword.get_values(:arg)
-    labels = meta |> Keyword.get_values(:label)
+
+    labels =
+      meta
+      |> Keyword.get_values(:label)
+      |> Enum.map(fn label ->
+        if label == nil do
+          "nil"
+        else
+          label
+        end
+      end)
```

Comment:
```suggestion
      |> Enum.map(&Atom.to_string/1)
```

Reply:
this does not work: 1st argument: not an atom

Reply:
hmm, isn't it because of https://github.com/membraneframework/unifex/pull/107/files#diff-e42f57944df3819667ac906a9e2d8bb2a04bb61a089cd070966dd1aa980c5a71R54?